release-tools/release-cc.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+script_name="${0##*/}"
+script_dir="$(dirname $(realpath -s "$0"))"
+release_tool="${script_dir}/release-tool"
+owner=${OWNER:-clearcontainers}
+
+function usage() {
+cat << EOT
+Usage: ${script_name}
+This script creates a new release for Clear Containers
+It tags and create release for:
+Proxy
```

Comment:
I'd leave a blank line before `Proxy` (and maybe indent these 3 words to make it clearer).